application/src/main/java/net/corda/v5/application/interop/parameters/TypedParameter.java

```diff
@@ -2,10 +2,19 @@
 
 import org.jetbrains.annotations.NotNull;
 
+/**
+ * A [TypedParameter] is a parameter having a name and a defined [ParameterType].
+ */
 public interface TypedParameter<T> {
+    /**
+     * @return The name of the parameter, e.g. "amount".
+     */
     @NotNull
     String getName();
 
+    /**
+     * @return The type of the parameter, e.g. [ParameterTypeLabel.STRING].
+     */
     @NotNull
     ParameterType<T> getType();
 
```

Comment:
Missing doc for `TypedParameterValue<T> of(T value);`.

Reply:
Wrap `ParameterTypeLabel` into a link, I don't know if we can do the same for  `ParameterTypeLabel.STRING`, but this latter is an example so it can remain unlinked.